spec/migrations/20160127210624_convert_configurations_to_settings_changes_spec.rb

```diff
@@ -43,6 +43,16 @@
         }
       )
 
+      _non_existing_tmpl_file = config_stub.create!(
+        :typ           => "hostdefaults",
+        :miq_server_id => 2,
+        :settings      => {
+          "values" => {
+            "string" => "hostdefaults value",
+          },
+        }
+      )
+
```

Comment:
This configurations row recreates the reported issue in #7571.  Note, there's nothing to assert here because we blindly skip this row.